src/lib/form-field/form-field-input.scss

```diff
@@ -89,6 +89,12 @@
     // happen, however it's possible to do it when clicking on a disabled input (see #13479).
     @include user-select(none);
 
+    &:-ms-input-placeholder {
+      // fix IE11 not able to focus programmatically with css style -ms-user-select: none
+      // see https://github.com/angular/material2/issues/15093
+      -ms-user-select: text;
```

Comment:
Actually on a second though, maybe we should set it to `auto`? That's the default value from the browser.

Reply:
@crisbeto I initially thought so, too, but [MSDN](https://msdn.microsoft.com/en-us/library/hh781492(v=vs.85).aspx) states that the initial value of `-ms-user-select` is `text` and `auto` is not an allowed property.

Reply:
Interesting. I guess that's why it has a vendor prefix.

Reply:
@crisbeto by the way, is there any chance to have a quick chat with you about your PR that adds a header to mat select? Please ping me on gitter.im. We have built a component that adds a search input field for mat select (see https://github.com/bithost-gmbh/ngx-mat-select-search), which has >10k downloads per month, and I would like to talk about integrating / building this feature in material itself.

Reply:
The reason my PR hasn't moved much in a long time is that we wanted to get the accessibility story for `mat-select` right before doing it, because we might've had to make some API changes. We should be able to get back to it once #14430 goes through.

Reply:
Thanks for the response. Is there any chance I could contribute a native material implementation of https://github.com/bithost-gmbh/ngx-mat-select-search?